smartsim/experiment.py

```diff
@@ -797,6 +796,56 @@ def reconnect_orchestrator(self, checkpoint: str) -> Orchestrator:
             logger.error(e)
             raise
 
+    def preview(
+        self,
+        output_format: t.Optional[str] = None,
+        output_filename: t.Optional[str] = None,
+        verbosity_level: t.Optional[str] = None,
+    ) -> str:
+        """Preview entity information prior to launch. This method
+        aggregates multiple pieces of information to give users insight
+        into what and how entities will be launched.  Any instance of
+        ``Model``, ``Ensemble`` or ``Orchestrator`` created by the
+        Experiment can be passed as an argument to the preview method.
+        param output_format: Set output destination. The possible accepted
```

Comment:
Missing `:` before `param` and `type`, the syntax should be `:param param_name:` and `:type param_name:` to avoid Sphinx errors or warnings.